modules/update-upgrading-web.adoc

```diff
@@ -22,18 +27,39 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to the correct
-`stable-<major_version>` channel for the version that you want to update to.
+. For production clusters, ensure that the *CHANNEL* is set to the correct
+`stable-*` channel for
+ifdef::within[]
+the version that you want to update to.
+endif::within[]
+ifdef::between[]
+your current minor version.
+endif::between[]
 +
 [IMPORTANT]
 ====
 For production clusters, you must subscribe to a `stable-*` channel.
 ====
-.. If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
 cluster.
-.. The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** The *DESIRED VERSION* indicates the cluster version that your cluster is running
 or is updating to.
 
-. Click *Updates Available*, select a version to update to, and click *Update*.
+. Click *Updates Available*, select
+ifdef::within[]
+a version to update to,
+endif::within[]
+ifdef::between[]
+the highest available version
+endif::between[]
+and click *Update*.
+The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
+the Operator upgrades on the *Cluster Operators* tab.
+
+ifdef::between[]
+. After the update completes, change the *CHANNEL* to the `stable-*` channel for the next minor version.
```

Comment:
"...and wait until the cluster-version operator refreshes the upgrade graph."?  It would be nice if we were watching the ClusterVersion object and triggering a fresh fetch when the channel changed (and maybe we are), but even then, it might take some time to pull the graph, or the network could be down, or whatever.  I think it's worth saying something to hint that into the reader's conciousness even if we don't take a big dive into all possible complications ;).